frontend/encore/simple-example.rst

```diff
@@ -317,6 +317,11 @@ split to *separate* files by Encore. Then, those files won't be downloaded until
 the moment a matching element (e.g. ``<div data-controller="lazy-example">``)
 appears on the page!
 
+.. note::
+
+    If you write your controllers using TypeScript, make sure
+    ``removeComments`` is not set to ``true`` in your TypeScript config.
```

Comment:
Maybe extend the sentence to explain the "why"?

Reply:
Not sure if this is necessary? I mean, the config is a comment, so it makes sense without explaining it? Or do you mean to explain that TypeScript is compiled before stimulusFetch config is handled by Simulus bridge?

Reply:
I like the note as is :)